src/SFA.DAS.AssessorService.Application.Api/Controllers/ScheduleController.cs

```diff
@@ -15,10 +15,10 @@ public ScheduleController(IScheduleRepository scheduleRepository)
             _scheduleRepository = scheduleRepository;
         }
 
-        [HttpGet("api/v1/schedule", Name="GetNextScheduleToRunNow")]
-        public async Task<IActionResult> GetNextScheduleToRunNow(int scheduleType)
+        [HttpGet("api/v1/schedule", Name = "GetScheduleRun")]
+        public async Task<IActionResult> GetScheduleRun(Guid scheduleRunId)
         {
-            var scheduleRun = await _scheduleRepository.GetNextScheduleToRunNow(scheduleType);
+            var scheduleRun = await _scheduleRepository.GetScheduleRun(scheduleRunId);
```

Comment:
Looks like you'rve removed "GetNextScheduleToRunNow"? I know it's clumsily named but it doesn't deserve to die. ;)

Reply:
Ah yes, this is because this function now grabs the details of the specified `scheduleRunId` rather than its type. The reason behind that is to give the caller more control over what they would like to view/modify